packages/client/hmi-client/src/components/models/Model.vue

```diff
@@ -75,15 +75,35 @@
 				</AccordionTab>
 				<AccordionTab header="State variables">
 					<DataTable
-						:value="model?.content.S"
+						:value="model?.content?.species"
```

Comment:
I think that this is coming from `petrinet-service.ts` and now needs to be changed there too.  May be worth using the typescript generation tool on this?